cranelift/assembler-x64/src/imm.rs

```diff
@@ -0,0 +1,209 @@
+//! Immediate operands to instructions.
+
+#![allow(clippy::module_name_repetitions)]
+#![allow(unused_comparisons)] // Necessary to use maybe_print_hex! with `u*` values.
+#![allow(clippy::cast_possible_wrap)] // Necessary to cast to `i*` for sign extension.
+
+use crate::api::{KnownOffset, KnownOffsetTable};
+use arbitrary::Arbitrary;
+
+/// This helper function prints the hexadecimal representation of the immediate
+/// value, but only if the value is greater than or equal to 10. This is
+/// necessary to match how Capstone pretty-prints immediate values.
```

Comment:
Egads -- is there a way to configure Capstone to always print in decimal or hex? (Necessary evil otherwise, not your fault, just ugly)

Reply:
I don't think so... it all seems to be hidden away behind a variant in [`ArchSyntax`](https://docs.rs/capstone/latest/capstone/arch/x86/enum.ArchSyntax.html). I suspect there's more configurability possible with a more x86-specific tool like XED but we're just not ready for that yet. The upside to the fuzzing approach here, though, is that the minute we switch to another disassembler that doesn't have this representation we will see fuzz bugs and be able to remove this code.

Reply:
Yep, sounds good, it's at least well-tested!